doi-transformer/src/main/java/no/unit/nva/doi/transformer/CrossRefConverter.java

```diff
@@ -277,14 +288,16 @@ private String filterIssnsByType(CrossRefDocument crossRefDocument, Isxn.IsxnTyp
             case JOURNAL_ARTICLE:
                 return createJournalArticle(document);
             case BOOK:
-                return createBookAnthology(document);
+            case BOOK_CHAPTER:
```

Comment:
This switch does the same as the previous switch, I guess these can be refactored to one.

Reply:
Back to if...., could possibly be joined with the other one as, to be used in the start of the converting..